README.md

```diff
@@ -88,4 +88,56 @@ To run the end to end tests you will need
 4. To run the e2e tests >> poetry run pytest todo_app/tests_e2e
 
 
+Running within Vagrant VM
+
+This project runs Vagrant on Hyper-V please follow these instructions for setup https://techcommunity.microsoft.com/t5/virtualization/vagrant-and-hyper-v-tips-and-tricks/ba-p/382373
```

Comment:
Minor, but worth knowing that I'm set up with virtual box & this works out of the box for that too 👍